doc/templates/uno/styles/main.css

```diff
@@ -3,6 +3,10 @@
+@media(min-width: 1024px){.visibility-opacity-transition,#header-container nav>ul>li.menu-item-has-children>ul ul,#header-container nav>ul>li.menu-item-has-children>ul{visibility:hidden;opacity:0;transition:visibility 100ms,opacity 100ms ease-in-out;list-style:none}}#header-container{padding:16px 32px;z-index:9999;background:#fff;display:flex;position:fixed;width:100%;min-height:67px;max-height:67px;border-bottom:1px solid #f4f2f3;justify-content:space-between}#header-container::after,#header-container::before{content:none}#header-container #menu-menu-principal{float:right;list-style:none}@media(max-width: 1023px){#header-container #menu-menu-principal{width:100%;max-height:70vh;overflow-y:auto}}@media(min-width: 1024px){#header-container #menu-menu-principal{display:flex}}#header-container #header-logo-container{width:36px;vertical-align:middle;z-index:9}#header-container #header-logo-container img{width:100%}@media(min-width: 1024px){#header-container #header-logo-container{min-width:120px}}#header-container .navbar-toggle{padding:0;margin:10px 0;display:none !important;z-index:9}#header-container .navbar-toggle .icon-bar{background:#333;width:20px}#header-container .navbar-toggle .icon-bar+.icon-bar{margin-top:3px}@media(max-width: 1023px){#header-container .navbar-toggle{display:block !important}}#header-container .navbar-brand{height:auto}#header-container nav a{padding:10px;font-size:13px;font-weight:600;font-stretch:normal;font-style:normal;line-height:14px;letter-spacing:normal;text-transform:uppercase;color:#242424}#header-container nav a:hover,#header-container nav a:focus{color:#7a67f8}
+@media(min-width: 1024px){#header-container nav a{font-size:14px;line-height:17px;padding: 0.2rem !important;}}
```

Comment:
@marcoscodnet, I would normally expect updates to `main.scss` and `main.css.map` in addition to `main.css` (similar to this previous PR: [#17165](https://github.com/unoplatform/uno/pull/17165/files)).

![image](https://github.com/user-attachments/assets/68ddee6e-e18d-412b-8697-95e88df26b34)

> [!IMPORTANT]
> Please make sure to build the intermediary assets locally for your changes using `npm run build` (similar to what is done in the [CI build for the PRs](https://github.com/unoplatform/uno/blob/master/build/ci/templates/docfx-intermediary-assets.yml)).